src/Htmxor/Http/HtmxContext.cs

```diff
@@ -1,16 +1,47 @@
-﻿using Microsoft.AspNetCore.Http;
+﻿using Htmx;
+using Microsoft.AspNetCore.Http;
 
 namespace Htmxor.Http;
 
-public class HtmxContext
+/// <summary>
+/// Encapsulates all HTMX-specific information related to an individual HTTP request
+/// </summary>
+public class HtmxContext : HtmxRequestHeaders
```

Comment:
Extending HtmxRequestHeaders will makes the following additional properties available:
```csharp
    /// <summary>
    /// the current URL of the browser
    /// </summary>
    public string CurrentUrl { get; }
    /// <summary>
    /// true if the request is for history restoration after a miss in the local history cache
    /// </summary>
    public bool HistoryRestoreRequest { get; }
    /// <summary>
    /// the user response to an hx-prompt on the client
    /// </summary>
    public string Prompt { get; }
    /// <summary>
    /// the id of the target element if it exists
    /// </summary>
    public string Target { get; }
    /// <summary>
    /// the name of the triggered element if it exists
    /// </summary>
    public string TriggerName { get; }
    /// <summary>
    /// the id of the triggered element if it exists
    /// </summary>
    public string Trigger { get; }
        
    /// <summary>
    /// Determines if the request was Boosted looking at "Hx-Boosted" header
    /// See https://htmx.org/docs/#boosting
    /// </summary>
    public bool Boosted { get; }
```